### PR TITLE
Feat/workspaces databases

### PR DESCRIPTION
**Pull request by cheikhgwane**
Workspace database view 

## Changes

Please list / describe the changes in the codebase for the reviewer(s).

- Add a page for viewing workspace database table. and shared tables.

## How/what to test

A few words about what needs to be tested, along with specific testing instructions if needed.

## Screenshots / screencast

image

### CODE REVIEW

File: src/workspace/layouts/WorkspaceLayout/Sidebar.tsx [hunk 1]
```diff
@@ -119,10 +119,10 @@ const Sidebar = (props: SidebarProps) => {
               {t("Files")}
             </NavItem>
             <NavItem
-              href={`/workspaces/${encodeURIComponent(workspaceId)}/databases`}
+              href={`/workspaces/${encodeURIComponent(workspaceId)}/database`}
```
Comment (qgerome) [inline src/workspace/layouts/WorkspaceLayout/Sidebar.tsx:122]: Better to stay with plurals "/databases"

File: src/pages/workspaces/[workspaceId]/database/index.tsx [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+              {(value) => <div>{`${value} row(s)`}</div>}
+            </BaseColumn>
+            <DateColumn
+              className="py-3"
+              relative
+              accessor="updatedAt"
+              id="updatedAt"
+              label="Last modified"
+            />
+            <BaseColumn className="cursor-pointer py-3">
```
Comment (qgerome) [inline src/pages/workspaces/[workspaceId]/database/index.tsx:157]: You can just use a "ChevronLinkColumn`

**Review by qgerome — CHANGES_REQUESTED** [review]

File: src/workspace/layouts/WorkspaceLayout/Sidebar.tsx (hunk 1 above)
Comment (cheikhgwane) [reply src/workspace/layouts/WorkspaceLayout/Sidebar.tsx:122]: Okay.

File: src/pages/workspaces/[workspaceId]/database/index.tsx (hunk 2 above)
Comment (cheikhgwane) [reply src/pages/workspaces/[workspaceId]/database/index.tsx:157]: Got it.

**Comment by cheikhgwane** [comment]
image
Updated @qgerome

**Pull request closed without merging**